spec/spec_helper.rb

```diff
@@ -22,6 +22,10 @@
   # rspec-expectations config goes here. You can use an alternate
   # assertion/expectation library such as wrong or the stdlib/minitest
   # assertions if you prefer.
+
+  # only runs tests with " , focus: true   "
+  # config.filter_run focus: true
+
```

Comment:
Is there a reason this is commented out?

Reply:
yes